lib/sidekiq_adhoc_job/web/job_presenter.rb

```diff
@@ -0,0 +1,59 @@
+# References: https://github.com/moove-it/sidekiq-scheduler/blob/master/lib/sidekiq-scheduler/job_presenter.rb
```

Comment:
actually what are the considerations when deciding which namespace to use uh? Like what made you move things out from `/services` and decide that it should be in the root directory or `/web` etc

Reply:
For the case of `JobPresenter`, its existence is to be the encapsulation of the data we want to display on the admin page, so I think it should belong to the `web` namespace.

As for the `WorkerFilesLoader`, I find it isn't really a business logic service (something that affects the application logic), it is more like a setup kind of class to load all the worker files and store it in memory. That's why I decided to move it out of `services`, and put it at the same level as `Web`, which is another setup kind of class.

Reply:
understooded! thank you 😁 👍